src/pages/AccountAddress.vue

```diff
@@ -142,12 +142,13 @@ export default {
                     decimalIndex + 5,
                 )}`;
             }
-            return `${strBalance} TLOS`;
+            // return `${strBalance} TLOS`;
+            return this.$('pages.tlos_balance', { balance: strBalance });
         },
         getAddressNativeExplorerURL() {
             if (!this.telosAccount) return '';
 
-            return `${process.env.NETWORK_EXPLORER}/account/${this.telosAccount}`;
+            return this.$('pages.account_url', { domain: process.env.NETWORK_EXPLORER, account: this.telosAccount });
```

Comment:
should we be using i18n for URLs? does this mean the expectation is that all URLs will now be localized? I don't think that's necessary

Reply:
Yes, @karynemayer suggested we should leave no string behind. Even if it does not need translation like urls or balance symbols (see `tlos_balance: '{ balance } TLOS'`)